platforms/cpuinfo.go

```diff
@@ -43,6 +45,32 @@ func cpuVariant() string {
 	return cpuVariantValue
 }
 
+// get CPU Architecture from uname -m
```

Comment:
- Please follow https://go.dev/doc/comment
- Mentioning `uname -m` is misleading since you are actually calling the underlying syscall, not uname command.

Reply:
Comment updated.